portal-impl/src/custom-sql/portal.xml

```diff
@@ -967,6 +967,35 @@
 				(ResourceTypePermission.roleId = ?)
 		]]>
 	</sql>
+	<sql id="com.liferay.portal.kernel.service.persistence.RoleFinder.countByGroupRoleAndTeamRole">
+		<![CDATA[
+			SELECT
+				COUNT(DISTINCT Role_.roleId) AS COUNT_VALUE
```

Comment:
Nope, I misunderstood how distinct worked. I'll send you an updated pull without the distinct.